src/services/resolve/services/scopeResolverService.js

```diff
@@ -0,0 +1,135 @@
+const { authenticate } = require('@feathersjs/authentication');
+const { equal } = require('../../../helper/compare').ObjectId;
+
+// get an json api conform entry
+const getDataEntry = ({
+	type, id, name, authorities = ['can-read'], attributes = {},
+}) => ({
+	type,
+	id,
+	attributes: Object.assign({}, {
+		name,
+		authorities,
+	}, attributes),
+});
+
+// get scopes from user object Id
+const equalId = (baseId) => (id) => equal(baseId, id);
+
+class ScopeResolver {
+	async get(id, params) {
+		const userService = this.app.service('/users');
+		const courseService = this.app.service('/courses');
+		const classService = this.app.service('/classes');
+		const teamService = this.app.service('/teams');
+
+		const response = {
+			links: {
+				self: '',
+				first: '',
+				last: '',
+				next: '',
+				prev: '',
+			},
+			data: [],
+		};
+
+		const user = await userService.get(id);
+		const userId = user._id;
+
+		response.data.push(getDataEntry({
+			type: 'user',
+			id: userId,
+			name: user.fullName,
+			authorities: [
+				'can-read',
+				'can-write',
+				'can-send-notifications',
+			],
+		}));
+
+		const [courses, classes, teams] = await Promise.all([
+			courseService.find({
+				query: {
+					$limit: 1000,
```

Comment:
however we want to get all users courses, classes and teams, this call should be wrapped into a chunked request that loops limited pages by itself.